src/Writer/XLSX/Options.php

```diff
@@ -63,4 +70,22 @@ public function getMergeCells(): array
     {
         return $this->MERGE_CELLS;
     }
+
+    public function setRowAttributes(Row $row, RowAttributes $attributes): Row
```

Comment:
No fluent interfaces please
```suggestion
    public function setRowAttributes(Row $row, RowAttributes $attributes): void
```